web-ui/src/main/resources/catalog/js/admin/UserGroupController.js

```diff
@@ -704,8 +704,12 @@
         $scope.groupUpdated = true;
       };
 
-      loadGroups();
-      loadUsers();
+      $scope.$watch('user', function(n, o) {
+        if (n !== undefined) {
```

Comment:
I loaded the page and first time `n={}`, no real session info, but as it's different than `undefined` enters in the `if` code. It's true that right after enters again in the `watch` and then `n`  has the session info. Maybe check with apart of undefined if contains any property like `n.profile` that is expected to be there.

Would be good also to check `n != o` to avoid multiple times loading the users/groups.